src/components/content/dropdowns/MessageDropdownItem.js

```
@@ -254,8 +271,19 @@ const MessageDropdownItem = ({ message, onSelect }) => {
                   {messageData?.timestamp ? formatTimestampToHowLongAgo(messageData.timestamp) : ''}
                 </div>
               </div>
+              
               <p className="text-neutral-50/80 whitespace-pre-wrap mb-3 text-sm leading-relaxed group-hover:text-neutral-50/90 transition-colors duration-200">
-                {messageData?.content}
+                {messageData?._matches?.content || messageData?._matches?.title
+                  ? highlightText(
+                      getTextWithMatchContext(
+                        messageData?.content,
+                        (messageData?._matches?.content || messageData?._matches?.title).term,
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)